solr_helpers/__init__.py

```diff
@@ -384,6 +384,8 @@ def build_solr_facets(attrs, filter_tags=None, include_nulls=True, unique=None,
 #
 def build_solr_query(filters, comb_with='OR', with_tags_for_ex=False, subq_join_field=None,
                      search_child_records_by=None):
+
+    # subq_join seems to be unused in IDC??
     ranged_attrs = Attribute.get_ranged_attrs()
```

Comment:
It isn't for the moment due to collapsed record style, can't guarantee it never will.